CONTRIBUTING.md

```diff
@@ -41,16 +41,12 @@ Each PR should pass the tests and the linter to be accepted.
 docker pull getmeili/meilisearch:latest # Fetch the latest version of MeiliSearch image from Docker Hub
 docker run -p 7700:7700 getmeili/meilisearch:latest ./meilisearch --master-key=masterKey --no-analytics=true
 go test -v ./...
-# Install golint if needed (see comment below)
-go get -u golang.org/x/lint/golint
-# Use golint
-golint
+# Use golangci-lint
+docker run --rm -v $(pwd):/app -w /app golangci/golangci-lint:v1.42.0 golangci-lint run -v
```

Comment:
Just a question why didn't use the `latest` instead of the `v1.42.0`?

Reply:
The CI step is also fixed to version 1.42.0. To mitigate differences in CI and local execution I fixed the version in the docker command to the same version

Reply:
Perfect